### PR TITLE
Examples of SDK usage

### PR DESCRIPTION
**Pull request by ekenigs**

### CODE REVIEW

**Review by mmaurello — APPROVED** [review]

**Review by mmaurello — APPROVED** [review]

File: packages/sdk/README.md [hunk 1]
```diff
@@ -1,3 +1,98 @@
...
+  { ethersSigner },
+);
+
+await send(min, (event) => console.log(event));
+```
+
+### Get balances
+
+```typescript
+const unsubscribe = await moonbeam.subscribeToAssetsBalanceInfo(
```
Comment (fxgamundi) [inline packages/sdk/README.md:68]: Just now coming to this realization, but maybe we should also offer a method to get balance of only one of the assets

Comment (ekenigs) [reply packages/sdk/README.md:68]: Yeah, I also thought about it.

Comment (ekenigs) [reply packages/sdk/README.md:68]: I think we should also move getMax from dapp to sdk.

File: packages/sdk/examples/simple/index.ts [hunk 2]
```diff
@@ -0,0 +1,113 @@
+import { AssetSymbol, ChainKey } from '@moonbeam-network/xcm-config';
+import { init, toDecimal } from '@moonbeam-network/xcm-sdk';
+import { Keyring } from '@polkadot/api';
+import { ethers } from 'ethers';
+import { setTimeout } from 'timers/promises';
+
+/**
+ * Add your moonbeam address and private key
+ */
+const moonbeamAddress = '';
```
Comment (fxgamundi) [inline packages/sdk/examples/simple/index.ts:10]: Maybe we can drop the addresses here, as we can get them from the private keys by doing `signer.address` and `polkadotAccount.address`

**Review by fxgamundi — APPROVED** [review]
LGTM

File: packages/sdk/README.md (hunk 1 above)
Comment (fxgamundi) [reply packages/sdk/README.md:68]: That's a good idea, should be useful to other teams

Comment (mmaurello) [reply packages/sdk/README.md:68]: agree

File: packages/sdk/examples/simple/index.ts (hunk 2 above)
Comment (ekenigs) [reply packages/sdk/examples/simple/index.ts:10]: I had some issues with that so I decided to do it dum way. Maybe we can change this in the furure.

**Pull request merged**